src/Common/src/System/IO/RowConfigReader.cs

```diff
@@ -19,6 +20,19 @@ public RowConfigReader(string buffer)
         {
             _buffer = buffer;
             _currentIndex = 0;
+            _comparisonKind = StringComparison.Ordinal;
+        }
+
+        /// <summary>
+        /// Constructs a new RowConfigReader which reads from the given string.
+        /// <param name="buffer">The string to parse through.</param>
+        /// <param name="comparisonKind">The comparison kind to use.</param>
+        /// </summary>
+        public RowConfigReader(string buffer, StringComparison comparisonKind)
```

Comment:
We aren't using it here, but since this file ended up in the "Common" directory I figured I ought to add the option to use a different StringComparison value if others pick up the code. I think "Ordinal" is an okay default for most parsing things that would use this, though.